srt-protocol/src/protocol/sender/mod.rs

```diff
@@ -353,7 +353,6 @@ impl Sender {
     }
 
     fn on_snd_event(&mut self, now: Instant) {
-        self.snd_timer.reset(now);
```

Comment:
Why no more reset?

Reply:
SND is a periodic timer. In SRT Live mode the SND period is calculated based on the max bandwidth limit and the average packet size. Resetting the timer here has the effect of lengthening the period which effectively lowers the max bandwidth limit. I was seeing an impact on throughput when I ran the high bandwidth test with extreme bitrates that pushed the limits of the implementation and/or my laptop (e.g. 800Mbit).

Reply:
Cool! Was just curious